utils/demo/helpers/addDropdownToToolbar.ts

```diff
@@ -3,33 +3,40 @@ export default function addDropDownToToolbar({
   options,
   container,
   onSelectedValueChange,
+  labelText,
 }: {
   id?: string;
   options: { values: number[] | string[]; defaultValue: number | string };
   container?: HTMLElement;
   onSelectedValueChange: (value: number | string) => void;
+  labelText?: string;
 }) {
   const { values, defaultValue } = options;
-  const select = document.createElement('select');
 
+  // Create label element if labelText is provided
+  if (labelText) {
+    const label = document.createElement('label');
+    label.htmlFor = id;
+    label.innerText = labelText;
+    container = container ?? document.getElementById('demo-toolbar');
```

Comment:
`container = container ?? document.getElementById('demo-toolbar');` can be moved up before `if(labelText)` and the same one on line 45 can be deleted.